.eslintrc

```diff
@@ -39,6 +39,18 @@
   },
   "ignorePatterns": [ "src/config.js", "src/config.*.js", "dist/*" ],
   "overrides": [
+    {
+      "files": [ "src/**" ],
+      "rules": {
+        "no-console": "error"
+      }
+    },
+    {
+      "files": [ "src/plugins/**" ],
+      "rules": {
+        "no-console": "off"
+      }
+    },
```

Comment:
These 2 entries make the rule `no-console` an error on all source files except the plugins.
The console should be used on the scripts, tests and plugins directly but we are changing to use only from the logger when inside the wallet routes.

I prefer setting `no-console` as off by default so any new directory is not affected by this rule.